pilot/pkg/networking/util/util.go

```diff
@@ -55,7 +55,8 @@ const (
 	// regarding the virtual service or destination rule used for each
 	IstioMetadataKey = "istio"
 	// The range of LoadBalancingWeight is [1, 128]
-	maxLoadBalancingWeight = 128
+	maxLoadBalancingWeight        = 128
+	IstioInboundInterceptionSplit = "sidecar.istio.io/inboundInterceptionSplit"
```

Comment:
wrong place.. lets follow pattern of other places with metadata usage

Reply:
I am a newbie in go playground. 
I do find a const var string has the prefix `sidecar.istio.io` in other files. 
You mean leave the function `IsInboundOutboundListenerSplit()` in this file as it's more about runtime config system widely but put the `IstioInboundInterceptionSplit` in other files?

Reply:
I think what you are proposing pilot should not be binded to k8s annotation style. 
Fixed in the later commits: xds metadata key is the string literal "IstioInboundInterceptionSplit" instead of "<namespace>/<key>"